tests/functional/api/groups/members_test.py

```diff
@@ -237,18 +237,23 @@ def authorization_header(auth_client) -> dict:
 
 
 class TestRemoveMember:
-    def test_it_removes_authed_user_from_group(self, app, db_session, factories):
-        group = factories.Group()
-        group_member = factories.User()
-        group.memberships.append(GroupMembership(user=group_member))
-        token = factories.DeveloperToken(user=group_member)
+    def test_it(self, app, db_session, factories):
+        user, other_user = factories.User.create_batch(size=2)
+        group, other_group = factories.Group.create_batch(size=2)
+        db_session.add_all(
+            [
+                GroupMembership(user=user, group=group),
+                GroupMembership(user=user, group=other_group),
+                GroupMembership(user=other_user, group=group),
+            ]
+        )
+        token = factories.DeveloperToken(user=user)
         db_session.add(token)
         headers = {"Authorization": "Bearer {}".format(token.value)}
         db_session.commit()
 
         app.delete("/api/groups/{}/members/me".format(group.pubid), headers=headers)
 
-        # We currently have no elegant way to check this via the API, but in a
-        # future version we should be able to make a GET request here for the
-        # group information and check it 404s
```

Comment:
It's better for the test assertions to work by querying the DB or ORM objects (as they already do) rather than making this test for the remove-member API also depend on some _other_ API call to work correctly in order for the assertions to succeed (a test should have only one reason to fail).